install/css/install.css

```diff
@@ -191,6 +191,10 @@ input[type="url"] {
 	width: 100%;
 }
 
+input[type="number"] {
```

Comment:
Should be `.elgg-input-number {`. We don't want to affect elements plugins may have added/styled manually.